tests/mock_crust_tests.rs

```diff
@@ -40,6 +40,10 @@
 extern crate itertools;
 #[macro_use]
 extern crate log;
+#[cfg_attr(feature="clippy", allow(useless_attribute))]
+#[allow(unused_extern_crates)]
+#[macro_use]
+extern crate maidsafe_utilities;
```

Comment:
Is that to make the tests compile with and without logging?
I usually just put the two lines
```rust
extern crate maidsafe_utilities;
let _ = maidsafe_utilities::log::init(false);
```
directly into the test function; that's not much more work that just the one line. Not sure about this …